install/include/classes/InstallDAO.class.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/************************************************************************/
+/* AChecker                                                             */
+/************************************************************************/
+/* Copyright (c) 2008 - 2011                                            */
```

Comment:
This new DAO class is your credit, @VictorAlagwu, please use your name as the author, and also fix the copyright to 2018. Thanks.

However, I noticed this class is quite similar with DAO.class.php. Is it possible to consolidate them into one shared class?